feature/eventdetail/build.gradle.kts

```diff
@@ -0,0 +1,86 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.android.library")
+    id("org.jetbrains.kotlin.plugin.parcelize")
+    alias(libs.plugins.kotlin.compose)
+    alias(libs.plugins.cash.paparazzi)
+}
+
+kotlin {
+    android()
+    jvm()
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(project(":core:displaymodels"))
+                implementation(project(":core:feature"))
+                implementation(project(":core:models")) // I still don't think this is necessary
```

Comment:
I'm not entirely against this, but it seems unnecessary because it's only for the ID type safe value class. Maybe we can put the value classes into their own module? Or we can revert this idea and only use it at the data layer? I'm not sure what's best.